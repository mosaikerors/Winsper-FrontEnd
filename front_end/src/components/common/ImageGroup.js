import React, {Component} from 'react';
import {StyleSheet, View, Dimensions} from 'react-native';
import {Image} from "react-native-elements";

const {width} = Dimensions.get("window");

export default class App extends Component{
    constructor(props){
        super(props);
    }
    render() {
        switch(this.props.length){
            case 1:
                return(
                    <View style={styles.container}>
                        <View style={styles.image}>
<<<<<<< HEAD
                            <Image source={{uri: this.props.images[0]}} style={styles.big}/>
=======
                            <Image source={{uri: this.props.images[0].uri}} style={styles.big}/>
>>>>>>> d999f2f1995e372e9380c12330fdebba83d60750
                        </View>
                    </View>);
                    
            case 2:
                return (
                    <View style={styles.container}>
                        <View style={styles.image}>
<<<<<<< HEAD
                            <Image source={{uri: this.props.images[0]}} style={styles.small}/>
                            <Image source={{uri: this.props.images[1]}} style={styles.small}/>
=======
                            <Image source={{uri: this.props.images[0].uri}} style={styles.small}/>
                            <Image source={{uri: this.props.images[1].uri}} style={styles.small}/>
>>>>>>> d999f2f1995e372e9380c12330fdebba83d60750
                        </View>
                    </View>);
            case 3:
                return (
                    <View style={styles.container}>
                        <View style={styles.image}>
<<<<<<< HEAD
                            <Image source={{uri: this.props.images[0]}} style={styles.small}/>
                            <Image source={{uri: this.props.images[1]}} style={styles.small}/>
                            <Image source={{uri: this.props.images[2]}} style={styles.small}/>
=======
                            <Image source={{uri: this.props.images[0].uri}} style={styles.small}/>
                            <Image source={{uri: this.props.images[1].uri}} style={styles.small}/>
                            <Image source={{uri: this.props.images[2].uri}} style={styles.small}/>
>>>>>>> d999f2f1995e372e9380c12330fdebba83d60750
                        </View>
                    </View>);
            case 4:
                return (
                    <View style={styles.container}>
                        <View style={styles.image}>
<<<<<<< HEAD
                            <Image source={{uri: this.props.images[0]}} style={styles.small}/>
                            <Image source={{uri: this.props.images[1]}} style={styles.small}/>
                        </View>
                        <View style={styles.image}>
                            <Image source={{uri: this.props.images[2]}} style={styles.small}/>
                            <Image source={{uri: this.props.images[3]}} style={styles.small}/>
=======
                            <Image source={{uri: this.props.images[0].uri}} style={styles.small}/>
                            <Image source={{uri: this.props.images[1].uri}} style={styles.small}/>
                        </View>
                        <View style={styles.image}>
                            <Image source={{uri: this.props.images[2].uri}} style={styles.small}/>
                            <Image source={{uri: this.props.images[3].uri}} style={styles.small}/>
>>>>>>> d999f2f1995e372e9380c12330fdebba83d60750
                        </View>
                    </View>);
            default:
                return null;
        }
    }
}
const styles = StyleSheet.create({
    container:{
        marginTop:5,
        marginBottom:5
    },
    image: {
        flex:0,
        flexDirection: 'row',
        justifyContent: 'flex-start',
    },
    big: {
        height:width/2,
        width:width/2
    },
    small: {
        height:width/4,
        width:width/4
    }
});