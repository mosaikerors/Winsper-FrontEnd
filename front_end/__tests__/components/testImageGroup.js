import React from 'react';
import renderer from 'react-test-renderer';
import ImageGroup from '../../src/components/hean/ImageGroup'

const images1 = [
    "../assents/images/p1.jpg"
];
const images2 = [
    "../assents/images/p1.jpg",
    "../assents/images/p2.jpg"
];
const images3 = [
    "../assents/images/p1.jpg",
    "../assents/images/p2.jpg",
    "../assents/images/p3.jpg"
];
const images4 = [
    "../assents/images/p1.jpg",
    "../assents/images/p2.jpg",
    "../assents/images/p3.jpg",
    "../assents/images/p4.jpg",
];

test('renders FilterItem correctly', () => {
    const tree = renderer.create(
        <ImageGroup images={images1} length={images1.length} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
test('renders FilterItem correctly', () => {
    const tree = renderer.create(
        <ImageGroup images={images2} length={images2.length} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});

test('renders FilterItem correctly', () => {
    const tree = renderer.create(
        <ImageGroup images={images3} length={images3.length} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});

test('renders FilterItem correctly', () => {
    const tree = renderer.create(
        <ImageGroup images={images4} length={images4.length} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
