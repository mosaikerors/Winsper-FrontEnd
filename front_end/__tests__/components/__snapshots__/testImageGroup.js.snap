// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FilterItem correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 5,
      "marginTop": 5,
    }
  }
>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p1.jpg",
          }
        }
        style={
          Object {
            "height": 375,
            "width": 375,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 375,
              "justifyContent": "center",
              "width": 375,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`renders FilterItem correctly 2`] = `
<View
  style={
    Object {
      "marginBottom": 5,
      "marginTop": 5,
    }
  }
>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p1.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p2.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`renders FilterItem correctly 3`] = `
<View
  style={
    Object {
      "marginBottom": 5,
      "marginTop": 5,
    }
  }
>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p1.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p2.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p3.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`renders FilterItem correctly 4`] = `
<View
  style={
    Object {
      "marginBottom": 5,
      "marginTop": 5,
    }
  }
>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p1.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p2.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p3.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "position": "relative",
        }
      }
    >
      <Image
        onLoadEnd={[Function]}
        source={
          Object {
            "uri": "../assents/images/p4.jpg",
          }
        }
        style={
          Object {
            "height": 187.5,
            "width": 187.5,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      />
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#bdbdbd",
              "height": 187.5,
              "justifyContent": "center",
              "width": 187.5,
            }
          }
          testID="RNE__Image__placeholder"
        />
      </View>
    </View>
  </View>
</View>
`;
